app/models/user.rb

```diff
@@ -4,8 +4,8 @@
 class User < ApplicationRecord
   require 'csv'
 
-  enum role: %i[admin_centro_inv admin_plataforma admin_patente
-                admin_cluster emprendedor_base usuario_general superadmin]
+  enum role: %i[admin_centro_inv administrador usuario_empresa
+                moderador usuario_emprendedor usuario_general superadmin]
```

Comment:
Falta agregar usuarios investigadores, y cambiaría los nombres para que estén más de acorde a lo que vimos con el cliente
```suggestion
  enum role: %i[centro_investigacion administrador empresa
                moderador emprendedor cluster investigador usuario_general superadmin]
```